spec/lib/hypgen/planner_spec.rb

```diff
@@ -13,12 +12,54 @@
       [
         { init_conf_tmp_id: 7, params: { experimentId: @experimentId, dap_token: Hypgen.config.dap_token } },
       ]}
+    allow(workflow).to receive(:start_time) {"2014-11-19T09:45:42.025Z"}
+    allow(workflow).to receive(:end_time) {"2014-11-20T09:45:42.025Z"}
```

Comment:
Space missing inside {.<br>Space missing inside }.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.